pkg/oc/cli/describe/helpers.go

```diff
@@ -340,8 +340,10 @@ func formatImageStreamTags(out *tabwriter.Writer, stream *imageapi.ImageStream)
 		}
 
 		switch {
-		case !hasSpecTag || tagRef.From == nil:
-			fmt.Fprintf(out, "  pushed image\n")
+		case !hasSpecTag:
```

Comment:
This isn't empty spec tag.  This is "no spec tag"

Reply:
Or "pushed image"

Reply:
updating here https://github.com/openshift/origin/pull/16198